carbon-design-system/carbon-platform
# fix(web-app): gracefully handle library ref not found

When getting asset related frameworks, if the library data was undefined/null, it is now excluded from the result set.

Also update carbon-components-vue's ref to be `vue2`

Verified by manual testing in local dev environment.

Steps to test
- Add a ref that doesn't exist to a random library from libraries.mjs
- Run the dev server
- Verify that the libraries page still loads, but the missing ref lib is excluded from the page.
